src/Chainweb/Version.hs

```diff
@@ -913,6 +914,14 @@ doCheckTxHash Development = (>= 110)
 doCheckTxHash (FastTimedCPM g) | g == petersonChainGraph = (>= 7)
 doCheckTxHash _ = const False
 
+-- | Placeholder name for 2.13 Pact changes
+chainweb213Pact :: ChainwebVersion -> BlockHeight -> Bool
+chainweb213Pact Mainnet01 = const False -- TODO
+chainweb213Pact Testnet04 = const False -- TODO
+chainweb213Pact Development = (>= 95)
+chainweb213Pact (FastTimedCPM g) | g == petersonChainGraph = (>= 5)
+chainweb213Pact _ = const False
```

Comment:
Why not make the  new behavior the default for all tests that don't explicitly require the old behavior?

Reply:
Hmm ... we've been doing `const False` up until now, probably because we're worried about genesis. Maybe it's best to change these across the board. 

How about leaving it `const False` for now and doing a dedicated PR for that?